shared/schemas/tabulatorTable.yml.json

```diff
@@ -1,94 +1,155 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://quiltdata.com/tabulator/config/1",
```

Comment:
@fiskus 
do we need these to have it working?

Reply:
let's add that back just in case

Reply:
I think, it's not required, but I'm not sure